doc/How-to-build.md

```diff
@@ -1,35 +1,72 @@
-# How to Compile
+# How to Build
 
 ## Windows 64-bit
+This guide is for building your Visual Studio 2022 solution for OrcaSlicer on Windows 64-bit.
 
 ### Tools Required
-- [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or Visual Studio 2019  
+- [Visual Studio 2022](https://visualstudio.microsoft.com/vs/) or Visual Studio 2019
+  ```shell
+  winget install --id=Microsoft.VisualStudio.2022.Professional -e
+  ```
 - [CMake (version 3.31)](https://cmake.org/) — **⚠️ version 3.31.x is mandatory**
+  ```shell
+  winget install --id=Kitware.CMake -v "3.31.6" -e
+  ```
 - [Strawberry Perl](https://strawberryperl.com/)
+  ```shell
+  winget install --id=StrawberryPerl.StrawberryPerl -e
+  ```
 - [Git](https://git-scm.com/)
+  ```shell
+  winget install --id=Git.Git -e
+  ```
+- [git-lfs](https://git-lfs.com/)
+  ```shell
+  winget install --id=GitHub.GitLFS -e
+  ```
+
+> [!Tip]
+>  - GitHub Desktop (optional): A GUI for Git and Git LFS, which already includes both tools.
+>    - ```shell
+>      winget install --id=GitHub.GitHubDesktop -e
+>      ```
 
 ### Instructions
-1. Clone the repository:
-   ```sh
-   git clone https://github.com/SoftFever/OrcaSlicer
-   ```
-2. Open the appropriate command prompt:
+  1. Clone the repository:
+     - If using GitHub Desktop clone the repository from the GUI.
+     - If using the command line:
+        1. Clone the repository:
+        ```shell
+        git clone https://github.com/SoftFever/OrcaSlicer
+        ```
+        2. Run lfs to download tools on Windows:
+        ```shell
+        git lfs pull
+        ```
+  2. Open the appropriate command prompt:
    - For Visual Studio 2019:  
      Open **x64 Native Tools Command Prompt for VS 2019** and run:
-     ```sh
+     ```shell
      build_release.bat
      ```
    - For Visual Studio 2022:  
      Open **x64 Native Tools Command Prompt for VS 2022** and run:
-     ```sh
+     ```shell
      build_release_vs2022.bat
      ```
+   3. If successful, you will find the VS 2022 solution file in:
```

Comment:
![image](https://github.com/user-attachments/assets/4b5edfd2-bab5-42e4-b582-ad9049338c89)

This line looks weird in preview. Not sure if it's github's issue though.